build-parent/pom.xml

```diff
@@ -277,6 +277,11 @@
                 <artifactId>quarkus-arc-test-supplement</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-arc-test-supplement-decorator</artifactId>
```

Comment:
:disappointed:

Reply:
I didn't figure any other way to add automated test. Ideas are welcome :shrug: 
That being said, I don't like it either and I should add that we have no such test for other similar cases either (such as archive with just interceptor binding or just qualifier and so on...).

Reply:
I know but I'm still disappointed :wink: ...